Assignment_5/Assignment_5_Problem_1.js

```diff
@@ -0,0 +1,37 @@
+//Assignment 5
+// Problem 1
+// Prolem statement 
+// http://www.codewars.com/kata/array-helpers
+
+// Solution
+
+
+Array.prototype.square = function() {
+    let squaredResult = this.map(x => x ** 2);
+    return squaredResult;
```

Comment:
This is good! Try this problem:- Modify the array directly instead of returning a new one.

Reply:
ya i did that actually first but then read the question carefully, it said we have to append a new array and return it!